paasta_tools/api/api.py

```diff
@@ -102,6 +101,7 @@ def setup_paasta_api():
 
 
 def main(argv=None):
+    monkey.patch_all()
```

Comment:
This should work, although the docs recommend doing it as soon as possible (ie at the top).  The itests run the api server "for real", so if the itests pass this should be fine